src/components/Generic/BaseAccordion.js

```diff
@@ -48,14 +48,14 @@ const SubTypeAccordion = ({
             && (
             <>
               <FocusedIcon
-                subType={subType}
+                subType={headerLabel}
```

Comment:
this `headerLabel` is also the same as `item.title`?

Reply:
yes, `item.title = headerLabel` is established in `dataArray`, line 30 of this file.

Reply:
Although it's already like this in `origin/master`, these should probably be a separate `Header` and `Content` components (could be in same file)

* function bodies would not be re-evaluated on every render
* more explicit
* less need to reason about the these variables that are closed-over


Also, I think if the following is done:
`renderContent={ResourceCards}` -- `ResourceCards` could get the props from each element of `dataArray`  (which should be an array of `{subType, resourceIds}` ?)